src/lib/openapi/spec/environments-project-schema.ts

```diff
@@ -6,6 +6,7 @@ export const environmentsProjectSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['version', 'environments'],
+    description: 'Environments defined for the current project',
```

Comment:
I'd switch this to something like this:

```suggestion
    description: 'Environments defined for a given project',
```

There's not necessarily a "current" project when you're working with APIs or reading specs, so I think this is clearer.